contracts/account-history-contract/src/states/history.rs

```diff
@@ -4,4 +4,4 @@ use cw_storage_plus::Map;
 use elys_bindings::account_history::types::PortfolioBalanceSnapshot;
 
 pub const HISTORY: Map<&str, HashMap<String, PortfolioBalanceSnapshot>> =
-    Map::new("history_portfolio_balance_snapshot_1");
+    Map::new("history_portfolio_balance_snapshot_2");
```

Comment:
should we add 
`pub const OLD_HISTORY: Map<&str, HashMap<String, PortfolioBalanceSnapshot>> =
    Map::new("history_portfolio_balance_snapshot_1");`
  ? just so that we know why we have `_2`

Reply:
it will not just add dead code:
```rust
pub const HISTORY: Map<&str, HashMap<String, PortfolioBalanceSnapshot>> =
    Map::new("history_portfolio_balance_snapshot_2");
#[allow(dead_code)]
const OLD_HISTORY: Map<&str, HashMap<String, PortfolioBalanceSnapshot>> =
    Map::new("history_portfolio_balance_snapshot_1");
```

Reply:
okay, makes sense